newton/_src/viewer/viewer_rerun.py

```
@@ -67,30 +69,56 @@ def __init__(
                 If False, the web viewer will only show the current frame to keep the memory usage constant when sending transform updates via :meth:`ViewerRerun.log_state`.
                 This is useful for visualizing long and complex simulations that would quickly fill up the web viewer's memory if the historical data was kept.
                 If True, the historical simulation data is kept in the viewer to be able to scrub through the simulation timeline. Defaults to False.
             keep_scalar_history (bool): If True, historical scala data logged via :meth:`ViewerRerun.log_scalar` is kept in the viewer.
             record_to_rrd (str): Path to record the viewer to a ``*.rrd`` recording file (e.g. "my_recording.rrd"). If None, the viewer will not record to a file.
+            server (bool | None): **Deprecated**. This parameter is no longer used.
+                                  The viewer now automatically handles connection setup.
+            launch_viewer (bool | None): **Deprecated**. Use `address=None` to spawn a viewer (default),
+                                         or provide an address to connect to an existing server.
         """
         if rr is None:
             raise ImportError("rerun package is required for ViewerRerun. Install with: pip install rerun-sdk")
```

Ruff (0.14.4)

Avoid specifying long messages outside the exception class

(TRY003)